.editorconfig

```diff
@@ -0,0 +1,7 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
Sure thing. I was afraid you would ask for that :)

> On Jan 11 2016, at 7:45 pm, Ari Porad &lt;notifications@github.com&gt;
wrote:  

>

> In [.editorconfig](https://github.com/shelljs/shelljs/pull/249#discussion_r4
9406493):

>  
>  
>     > @@ -0,0 +1,7 @@

>     > +# EditorConfig is awesome: http://EditorConfig.org

>  
>

> Can you move this into a separate PR? I'd like to try and keep them small
and focused. (I can show you some git-fu for how to do that if you'd like,
because I __totally__ understand git :P).

>

> —  
Reply to this email directly or [view it on GitHub](https://github.com/shelljs
/shelljs/pull/249/files#r49406493).![](https://github.com/notifications/beacon
/AC4eoB2krjypBzYqHYza3T4DsXX3aORLks5pZFIdgaJpZM4GcyGK.gif)